[IMP] productivity: calendar add troubleshooting (#8604)
odoo/documentation: content/applications/productivity/calendar/outlook.rst

.. seealso::
   - :doc:`/applications/general/integrations/mail_plugins/outlook`
   - :doc:`google`

Troubleshoot sync
=================

There may be times when the Microsoft Outlook calendar account does not sync correctly with Odoo.

> Sync issues can be seen in the logs of the database. In these cases the account needs

Reviewer: ```suggestion
Sync issues can be seen in the database logs. In these cases, the account needs
```

> There may be times when the Microsoft Outlook calendar account does not sync correctly with Odoo.
> Sync issues can be seen in the database logs. In these cases, the account needs troubleshooting. A
> reset can be performed using the :guilabel:`Reset Account` button, which can be accessed by
> navigating to :menuselection:`Settings app --> Manage Users`. Then select the user to modify the
> calendar on, and click on the :guilabel:`Calendar` tab.

Reviewer 2: ```suggestion
There may be times when the *Microsoft Outlook Calendar* account does not sync correctly with Odoo.
Sync issues can be seen in the database logs. 

In these cases, the account needs troubleshooting. A reset can be performed using the
:guilabel:`Reset Account` button, which can be accessed by navigating to :menuselection:`Settings
app --> Manage Users`. Then, select the user to modify the calendar, and click on the
:guilabel:`Calendar` tab.
```

.. image:: outlook/outlook-reset.png
   :align: center
   :alt: Reset buttons highlighted on the calendar tab of the user.

Next, click :guilabel:`Reset Account` under the correct calendar.

Reset options
-------------

> The following reset options are available for troubleshooting Microsoft Outlook calendar sync with

Reviewer 2: ```suggestion
The following reset options are available for troubleshooting *Microsoft Outlook Calendar* sync with
```

Reviewer 2: suggesting to capitalize *through* calendar because that's how Microsoft refers to it on their official website: https://support.microsoft.com/en-us/office/introduction-to-the-outlook-calendar-d94c5203-77c7-48ec-90a5-2e2bc10bd6f8

Odoo:

.. image:: outlook/reset-calendar.png
   :align: center
   :alt: Outlook calendar reset options in Odoo.

:guilabel:`User's Existing Events`:

 - :guilabel:`Leave them untouched`: no changes to the events.

>  - :guilabel:`Delete from the current Microsoft Calendar account`: delete the events from Microsoft
>    calendar.
>  - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
>  - :guilabel:`Delete from both`: delete the events from both: Microsoft calendar and Odoo calendar.

Reviewer 2: ```suggestion
 - :guilabel:`Delete from the current Microsoft Calendar account`: delete the events from *Microsoft
   Outlook Calendar*.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: *Microsoft Outlook Calendar* and Odoo calendar.
```

:guilabel:`Next Synchronization`:

>  - :guilabel:`Synchronize only new events`: sync new events on either/both: Microsoft calendar, Odoo
>    calendar.
>  - :guilabel:`Synchronize all existing events`: sync all events on either/both Microsoft calendar,
>    Odoo calendar.

Reviewer 2: ```suggestion
 - :guilabel:`Synchronize only new events`: sync new events on either/both: *Microsoft Outlook Calendar*, Odoo
   calendar.
 - :guilabel:`Synchronize all existing events`: sync all events on either/both: *Microsoft Outlook Calendar*,
   Odoo calendar.
```

>  - :guilabel:`Leave them untouched` (no changes to the events)
>  - :guilabel:`Delete from the current Microsoft Calendar account` (delete the events from Microsoft
>    calendar)
>  - :guilabel:`Delete from Odoo` (delete the events from the Odoo calendar)
>  - :guilabel:`Delete from both` (delete the events from both: Microsoft calendar and Odoo calendar)
> 
>  :guilabel:`Next Synchronization`:
> 
>  - :guilabel:`Synchronize only new events` (sync new events on either/both: Microsoft calendar, Odoo
>    calendar)
>  - :guilabel:`Synchronize all existing events` (sync all events on either/both Microsoft calendar,
>    Odoo calendar)

Reviewer: ```suggestion
 - :guilabel:`Leave them untouched`: no changes to the events.
 - :guilabel:`Delete from the current Microsoft Calendar account`: delete the events from Microsoft
   calendar.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: Microsoft calendar and Odoo calendar.

 :guilabel:`Next Synchronization`:

 - :guilabel:`Synchronize only new events`: sync new events on either/both: Microsoft calendar, Odoo
   calendar.
 - :guilabel:`Synchronize all existing events`: sync all events on either/both Microsoft calendar,
   Odoo calendar.
```

> Click :guilabel:`Confirm` after making the selection about modifying the user's events and the

Reviewer 2: ```suggestion
Click :guilabel:`Confirm` after making the selection to modify the user's events and the
```

> Click on :guilabel:`Confirm` after making the selection about modifying the user's events and the

Reviewer: ```suggestion
Click :guilabel:`Confirm` after making the selection about modifying the user's events and the
```

:guilabel:`Calendar` tab.

.. image:: outlook/outlook-reset.png
   :align: center
   :alt: Reset buttons highlighted on the calendar tab of the user.

Next, click :guilabel:`Reset Account` under the correct calendar.

Reset options
-------------

The following reset options are available for troubleshooting *Microsoft Outlook Calendar* sync with
Odoo:

.. image:: outlook/reset-calendar.png
   :align: center
   :alt: Outlook calendar reset options in Odoo.

:guilabel:`User's Existing Events`:

 - :guilabel:`Leave them untouched`: no changes to the events.
 - :guilabel:`Delete from the current Microsoft Calendar account`: delete the events from *Microsoft
   Outlook Calendar*.
 - :guilabel:`Delete from Odoo`: delete the events from the Odoo calendar.
 - :guilabel:`Delete from both`: delete the events from both: *Microsoft Outlook Calendar* and Odoo
   calendar.

:guilabel:`Next Synchronization`:

 - :guilabel:`Synchronize only new events`: sync new events on either/both: *Microsoft Outlook
   Calendar*, Odoo calendar.
 - :guilabel:`Synchronize all existing events`: sync all events on either/both: *Microsoft Outlook

>    Calendar*, Odoo calendar.

Reviewer 3: nested colons here, as well
